pkg/broker/provision_job.go

```diff
@@ -59,8 +56,8 @@ func (p *ProvisionJob) Run(token string, msgBuffer chan<- WorkMsg) {
 		return
 	}
 
-	p.log.Info("Destroying APB sandbox...")
-	sm.DestroyApbSandbox(podName, p.serviceInstance.Context.Namespace)
```

Comment:
It seemed unnecessary to have it here (the sandboxes are created in the apb module but destroyed in the broker module), especially since the logic for it was duplicated in `provision_job.go`, `deprovision_job.go` and `broker.go` (implemented for all 4 actions in there too, so provision and deprovision were actually implemented twice). 

Since I required the extra context to delete the sandboxes (`serviceInstance.Context.Namespace` is not always where the apb is running anymore), I figured I'd move this logic to the end of the action execution, where I think it makes more sense and where I can more closely track the effects of my changes.